src/labels_by_user_input.py

```diff
@@ -56,3 +68,17 @@ def labels_by_user_input(event_data, pull):
                 description=STATUS_DESCRIPTION_MISSING_MAINTAINERS_APPROVAL,
                 context=NEEDS_MAINTAINERS_APPROVE,
             )
+
+        semver_user_input = get_semver_user_input(body=body)
+        label_to_add = SEMVER_USER_INPUT_LABELS_MAP[semver_user_input]
+        if semver_user_input and label_to_add not in get_labels(pull):
```

Comment:
call `get_labels` once at the top and use it when needed.
it is been called several times here.